bedevere/util.py

```diff
@@ -1,9 +1,20 @@
+from asyncio import Future
 import enum
 import sys
 
 import gidgethub
 
+
+DEFAULT_BODY=""
+TAG_NAME = "issue-number"
 NEWS_NEXT_DIR = "Misc/NEWS.d/next/"
+CLOSING_TAG = f"<!-- /{TAG_NAME} -->"
+BODY = f"""\
+{{body}}
+<!-- {TAG_NAME}: gh-{{issue_number}} -->
+https://bugs.python.org/issue{{issue_number}}
```

Comment:
It should just be `gh-{{issue_number}}`, no links. GitHub will automatically link that in the UI.
We shouldn't have links to b.p.o anymore.

Reply:
Fixed in updated PR.